### ISSUE TITLE
Potential Bug in Content Loss

### ISSUE THREAD

**Issue by kktsubota** [photosynthesis-team/piq#258]
**Describe the bug**
If the length of `layers` mismatches `weights`, the content loss uses only the first layer **silently**.

**To Reproduce**

```python
import torch
import piq

img = torch.ones((1, 3, 256, 256))
ref_img = torch.zeros((1, 3, 256, 256))

piq.ContentLoss(layers=['relu1_2', 'relu2_2'])(img, ref_img)
# tensor(43.9598)
piq.ContentLoss(layers=['relu1_2'])(img, ref_img)
# tensor(43.9598)
piq.ContentLoss(layers=['relu1_2', 'relu2_2'], weights=[1.0, 1.0])
# tensor(214.3362)
```

**Expected behavior**
raise warnings or errors if the length of `layers` mismatches `weights`.
